### PR TITLE
[WiP] Segmentation

### PR DESCRIPTION
**Pull request by jamesmcclain**
1. [x] Complete or remove instance segmentation
2. [ ] Semantic segmentation
   - [ ] Deeplap
      - [x] Protobuf
      - [x] Configuration
         - [x] Backend configuration
         - [x] Workflow configuration
      - [ ] Label store
      - [ ] Task
      - [ ] Backend
      - [x] `compute_raster_stats`
      - [x] `make_training_chips`
      - [x] `train`
      - [ ] `predict`
      - [ ] `eval`
3. [ ] Ensure remote operation
4. [ ] Prediction packages
5. [ ] Unit tests
6. [ ] Review

### CODE REVIEW

**Comment by jamesmcclain** [comment]
As an initial step, labeled pixels are rendered in black (that will change later).

File: src/rastervision/ml_tasks/semantic_segmentation.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+from rastervision.core.ml_task import MLTask
+from rastervision.core.box import Box
+
+
+class SemanticSegmentation(MLTask):
+
+    def get_train_windows(self, scene, options):
+        raster_source = scene.raster_source
+        meta = raster_source.image_dataset.meta
```
Comment (lewfish) [inline src/rastervision/ml_tasks/semantic_segmentation.py]: Could use RasterSource.get_extent() instead

File: src/rastervision/ml_tasks/semantic_segmentation.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        width = meta.get('width')
+        height = meta.get('height')
+
+        label_store = scene.ground_truth_label_store
+
+        chip_size = options.chip_size
+
+        windows = []
+        for i in range(100):  # XXX insensitive
+            xmin = np.random.randint(low=0, high=width-chip_size)
```
Comment (lewfish) [inline src/rastervision/ml_tasks/semantic_segmentation.py]: Could use extent.make_random_square() instead.

File: src/rastervision/ml_tasks/semantic_segmentation.py (hunk 1 above)
Comment (jamesmcclain) [reply src/rastervision/ml_tasks/semantic_segmentation.py]: Thank you

File: src/rastervision/ml_tasks/semantic_segmentation.py (hunk 2 above)
Comment (jamesmcclain) [reply src/rastervision/ml_tasks/semantic_segmentation.py]: Thank you

**Comment by jamesmcclain** [comment]
Will reopen later.

**Pull request closed without merging**